src/main/java/com/postgraduate/domain/payment/application/usecase/PaymentManageUseCase.java

```diff
@@ -17,14 +21,22 @@
 @Transactional
 public class PaymentManageUseCase {
     private final PaymentSaveService paymentSaveService;
-    private final MentoringGetService mentoringGetService;
+    private final SeniorGetService seniorGetService;
+    private final UserGetService userGetService;
     private final SalaryGetService salaryGetService;
+    private final SalaryUpdateService salaryUpdateService;
+    private final static String SUCCESS = "0000";
 
-    public void savePay(PaymentResultRequest paymentResultRequest) {
-        long mentoringId = Long.parseLong(paymentResultRequest.PCD_PAY_GOODS());
-        Mentoring mentoring = mentoringGetService.byMentoringId(mentoringId);
-        Salary salary = salaryGetService.bySenior(mentoring.getSenior());
-        Payment payment = PaymentMapper.resultToPayment(mentoring, salary, paymentResultRequest);
+    public void savePay(PaymentResultRequest request) {
+        if (!request.PCD_PAY_CODE().equals(SUCCESS))
```

Comment:
null 이 아님이 보장되는 SUCCESS를 기준으로 비교하는 건 어떨까욤?
```suggestion
        if (!SUCCESS.equals(request.PCD_PAY_CODE()))
```

Reply:
만약 request에 PACD_PAY_CODE가 null이라면 순서가 어떻게 되던 npe 발생은 똑같지 않을까요?
근데 저 부분은 페이플에서 주는거라 값이 없을 가능성이 0에 가깝다고 생각해요

Reply:
참조하는 객체가 null이라면 NPE 가 발생하지만 인자는 null 이어도 NPE 가 발생하지 않는 걸로 알고있습니닷

`PCD_PAY_CODE`도 null 이 아니라는 게 보장되면 상관없겠네용!